server/token.go

```diff
@@ -37,8 +39,11 @@ const (
 func token(length int) string {
 	result := ""
 	for i := 0; i < length; i++ {
-		x := rand.Intn(len(SYMBOLS) - 1)
-		result = string(SYMBOLS[x]) + result
+		x, err := rand.Int(rand.Reader, big.NewInt(int64(len(SYMBOLS))))
+		if err != nil {
+			log.Fatal("Failed to generate token")
```

Comment:
we should return an error and gracefully fail the request with a reason to the user

Reply:
Can we just ignore err in this case?
I really doubt that we can get error in this case ever using rand.Reader
https://cs.opensource.google/go/go/+/master:src/crypto/rand/util.go;l=85
And we defenetly won't  panic there

Reply:
digging further, that's on unix system:
https://cs.opensource.google/go/go/+/master:src/crypto/rand/rand_unix.go;l=62
https://cs.opensource.google/go/go/+/master:src/crypto/rand/rand_unix.go;l=72

It's not excluded 100% that they won't return an error.

Indeed I guess that on our docker image (that's stratch based) the first will always return an error: do you mind trying to build the docker image from your branch and test?

thanks!

Reply:
`
Indeed I guess that on our docker image (that's stratch based) the first will always return an error: do you mind trying to build the docker image from your branch and test?
`
I tried run scratch image with dropped all capabilites, and it works. But i can do something like fallback

Reply:
I can reproduce and this seems to be the reason:
```
// On Linux, FreeBSD, Dragonfly, NetBSD and Solaris, Reader uses getrandom(2) if
// available, /dev/urandom otherwise.
```

but, please, note:
```
// On other Unix-like systems, Reader reads from /dev/urandom.
```

so I guess on other Unix-like systems it won't work.

not sure how to test

Reply:
`I can reproduce and this seems to be the reason:`
how?

`so I guess on other Unix-like systems it won't work.`
on other unix-like system should be /dev/urandom, but what another unix-like system? HP-UX may be?

Reply:
> `I can reproduce and this seems to be the reason:` how?
> 
> `so I guess on other Unix-like systems it won't work.` on other unix-like system should be /dev/urandom, but what another unix-like system? HP-UX may be?

let's check the error and panic with a message with the link to this PR

Reply:
> > `I can reproduce and this seems to be the reason:` how?
> > `so I guess on other Unix-like systems it won't work.` on other unix-like system should be /dev/urandom, but what another unix-like system? HP-UX may be?
> 
> let's check the error and panic with a message with the link to this PR

What you mean? How?

Reply:
> What you mean? How?

nevermind!

see my last suggestion please :)